lib/travis/build/script/shared/jdk.rb

```diff
@@ -49,7 +44,9 @@ def setup
         end
         
         def disable_gradle_daemon
-          sh.cmd 'mkdir -p ~/.gradle && echo "org.gradle.daemon=false" >> ~/.gradle/gradle.properties', echo: false, timing: false
+          sh.if '"$TRAVIS_DIST" == precise || "$TRAVIS_DIST" == trusty' do
```

Comment:
Why not on Bionic or newer?

The Bionic LTS release was released April 2018, half a year before this PR was made:
https://wiki.ubuntu.com/Releases